source/base/job_identifier.cc

```diff
@@ -20,7 +20,7 @@
 # include <unistd.h>
 #endif
 
-#ifdef DEAM_II_MSVC
+#ifdef DEAL_II_MSVC
 #  include <WinSock2.h>
```

Comment:
Yes. I grepped for MSVC and saw the typo but didn't think about the fact that this include is useless.